### PR TITLE
remove the klusterlet when cleaning spoke resources

### PR DESCRIPTION
**Pull request by haoqing0110**
Fix: https://issues.redhat.com/browse/ACM-12646 
Ref: https://github.com/stolostron/rhacm-docs/pull/5449

### CODE REVIEW

**Comment by haoqing0110** [comment]
@oafischer could you help take a look? this needs to be cherry-picked to the previous release version doc as well.

**Review by oafischer — APPROVED** [review]
LGTM

**Review by swopebe — APPROVED** [review]

File: troubleshooting/trouble_already_exists.adoc [hunk 1]
```diff
@@ -30,6 +30,12 @@ oc get all -n open-cluster-management-agent-addon
 [#resolving-problem-already-exists-ocp-import]
 == Resolving the problem: Already exists when importing {ocp-short} cluster
 
+Remove the `klusterlet` custom resource by using the following command:
+
+----
+oc get klusterlet | grep klusterlet | awk '{print $1}' | xargs oc patch klusterlet --type=merge -p '{"metadata":{"finalizers": []}}'
+----
```
Comment (dockerymick) [inline troubleshooting/trouble_already_exists.adoc]:
```suggestion
Remove the `klusterlet` custom resource by using the following command:

[source,bash]
----
oc get klusterlet | grep klusterlet | awk '{print $1}' | xargs oc patch klusterlet --type=merge -p '{"metadata":{"finalizers": []}}'
----
```

**Comment by dockerymick** [comment]
With two approvals, this PR can be merged. Thanks @haoqing0110

**Pull request merged**